src/components/structures/RoomView.js

```diff
@@ -234,10 +296,6 @@ module.exports = React.createClass({
                 var callState;
 
                 if (call) {
-                    // Call state has changed so we may be loading video elements
-                    // which will obscure the message log.
-                    // scroll to bottom
-                    this.scrollToBottom();
```

Comment:
It just didn't seem to make much sense. I don't want to be scrolled to the bottom when a call arrives. I kinda agree it's a separate problem, though, so if you insist I'll make it the subject of a separate PR...